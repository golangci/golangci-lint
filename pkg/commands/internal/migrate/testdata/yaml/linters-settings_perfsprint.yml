issues:
  # Only to not generate unrelated elements inside golden.
  exclude-use-default: false
  # Only to not generate unrelated elements inside golden.
  exclude-generated: strict
  # Only to not generate unrelated elements inside golden.
  exclude-dirs-use-default: false

linters-settings:
  perfsprint:
    integer-format: false
    int-conversion: false
    error-format: false
    err-error: true
    errorf: false
    string-format: false
    sprintf1: false
    strconcat: false
    bool-format: false
    hex-format: false
