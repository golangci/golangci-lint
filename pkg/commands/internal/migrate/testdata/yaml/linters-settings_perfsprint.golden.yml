version: "2"
linters:
  settings:
    perfsprint:
      integer-format: false
      int-conversion: false
      error-format: false
      err-error: true
      errorf: false
      string-format: false
      sprintf1: false
      strconcat: false
      bool-format: false
      hex-format: false
